From patchwork Fri Feb 22 08:10:09 2019
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Leo Yan <leo.yan@linaro.org>
X-Patchwork-Id: 10825413
Return-Path: <kvm-owner@kernel.org>
Received: from mail.wl.linuxfoundation.org (pdx-wl-mail.web.codeaurora.org
 [172.30.200.125])
	by pdx-korg-patchwork-2.web.codeaurora.org (Postfix) with ESMTP id 2F0866C2
	for <patchwork-kvm@patchwork.kernel.org>;
 Fri, 22 Feb 2019 08:10:30 +0000 (UTC)
Received: from mail.wl.linuxfoundation.org (localhost [127.0.0.1])
	by mail.wl.linuxfoundation.org (Postfix) with ESMTP id 1FC6E318F6
	for <patchwork-kvm@patchwork.kernel.org>;
 Fri, 22 Feb 2019 08:10:30 +0000 (UTC)
Received: by mail.wl.linuxfoundation.org (Postfix, from userid 486)
	id 130F13190E; Fri, 22 Feb 2019 08:10:30 +0000 (UTC)
X-Spam-Checker-Version: SpamAssassin 3.3.1 (2010-03-16) on
	pdx-wl-mail.web.codeaurora.org
X-Spam-Level: 
X-Spam-Status: No, score=-4.7 required=2.0 tests=BAYES_00,DKIM_SIGNED,
	DKIM_VALID,DKIM_VALID_AU,MAILING_LIST_MULTI,RCVD_IN_DNSWL_HI,RCVD_IN_SBL_CSS
	autolearn=ham version=3.3.1
Received: from vger.kernel.org (vger.kernel.org [209.132.180.67])
	by mail.wl.linuxfoundation.org (Postfix) with ESMTP id B75F2318F6
	for <patchwork-kvm@patchwork.kernel.org>;
 Fri, 22 Feb 2019 08:10:29 +0000 (UTC)
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S1726412AbfBVIKZ (ORCPT
        <rfc822;patchwork-kvm@patchwork.kernel.org>);
        Fri, 22 Feb 2019 03:10:25 -0500
Received: from mail-pg1-f193.google.com ([209.85.215.193]:40439 "EHLO
        mail-pg1-f193.google.com" rhost-flags-OK-OK-OK-OK) by vger.kernel.org
        with ESMTP id S1725927AbfBVIKY (ORCPT <rfc822;kvm@vger.kernel.org>);
        Fri, 22 Feb 2019 03:10:24 -0500
Received: by mail-pg1-f193.google.com with SMTP id u9so771528pgo.7
        for <kvm@vger.kernel.org>; Fri, 22 Feb 2019 00:10:24 -0800 (PST)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
        d=linaro.org; s=google;
        h=from:to:cc:subject:date:message-id;
        bh=0454jgmqadKslS6AXiSTPhnxq89hNMaMPVBArz7LJTA=;
        b=spQDfxPwxJ5BmPJmXZs7ysS4KIf5umW36JKTQlSRLxtVsrp7xtULyvKqCkeTBI/rsN
         sY6SCMGvCLCHVYebTEVvopungAuSDEZgCQCTVHHt3Ez3Czi3LpJKMPBYJL15TKO8MF4M
         ogoyg9OhuX0MEzfcMwuqwTWbKyjBYJ+p4sKu+kYLIbVZqfA/3OgzMcQvqcRgUV/bMWi1
         HdGwNfDU7zbPFIK4JoZh3NwOD+ROtMLhq7UWv/S0iN02fJAgnsUJbtjrchfHNqx74F28
         Vzj5BDehXe8WANOnDN0JAt8F4y30BTINSBofdTAvUlRa03GXVd/nkfybM2ui4EH4Z5J8
         HBdg==
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
        d=1e100.net; s=20161025;
        h=x-gm-message-state:from:to:cc:subject:date:message-id;
        bh=0454jgmqadKslS6AXiSTPhnxq89hNMaMPVBArz7LJTA=;
        b=NBdL8pV+t+k6moH4zKGEL6wXo5Ud1y2Dvzpx+QlZKFsnxbBSynrHeabs97w2hC2s6s
         RTqRTzK2dz907l+uVKgo8KpCnQppK83Cv4TdbHAONW/UruWBq09XK/23k+edeQAk2zzP
         yzwBw2qdJ6+bcJLhOUDevl/dNdB8eimAsMro+htkN4kqBHL3TDgaa8My0snXMIna++3x
         5m2RHZ1AdInN9CjlNx5H072sQ3HsaWlXCptQv8K2MMh9GnTMaGqI1vikgWMFrL8XKSRR
         7BaQPryzwfsBFK2KZxO0tg7IhDRNxSn5MfDxdsPJajdpN6WAtYupNDhSJW+60nc+nGAC
         sBLA==
X-Gm-Message-State: AHQUAuaVwwNxmU/Fhq8B7aUu8LiwxqVGAIWYrzMAr5mSRA92ed46IIB0
        GgsTWlb53tVg4dT+eSyqfLZuUQ==
X-Google-Smtp-Source: 
 AHgI3IZB87IofKx0ZDqJQQkz/AZqvtfwKr3vfaP6hWKJASMkc8e+PaunqvCKaLZj6a8l43iysk2g6w==
X-Received: by 2002:a63:1204:: with SMTP id h4mr2782827pgl.51.1550823024091;
        Fri, 22 Feb 2019 00:10:24 -0800 (PST)
Received: from localhost.localdomain (li1195-158.members.linode.com.
 [45.79.96.158])
        by smtp.gmail.com with ESMTPSA id
 b80sm1341927pfc.20.2019.02.22.00.10.21
        (version=TLS1_2 cipher=ECDHE-RSA-AES128-GCM-SHA256 bits=128/128);
        Fri, 22 Feb 2019 00:10:23 -0800 (PST)
From: Leo Yan <leo.yan@linaro.org>
To: Paolo Bonzini <pbonzini@redhat.com>,
 =?utf-8?b?UmFkaW0gS3LEjW3DocWZ?= <rkrcmar@redhat.com>, kvm@vger.kernel.org,
 linux-kernel@vger.kernel.org
Cc: Leo Yan <leo.yan@linaro.org>
Subject: [PATCH] KVM: Minor cleanups for kvm_main.c
Date: Fri, 22 Feb 2019 16:10:09 +0800
Message-Id: <20190222081009.540-1-leo.yan@linaro.org>
X-Mailer: git-send-email 2.17.1
Sender: kvm-owner@vger.kernel.org
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org
X-Virus-Scanned: ClamAV using ClamSMTP

This patch contains two minor cleanups: firstly it puts exported symbol
for kvm_io_bus_write() by following the function definition; secondly it
removes a redundant blank line.

Signed-off-by: Leo Yan <leo.yan@linaro.org>
---
 virt/kvm/kvm_main.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/virt/kvm/kvm_main.c b/virt/kvm/kvm_main.c
index 2679e476b6c3..735e42896dda 100644
--- a/virt/kvm/kvm_main.c
+++ b/virt/kvm/kvm_main.c
@@ -3496,6 +3496,7 @@ int kvm_io_bus_write(struct kvm_vcpu *vcpu, enum kvm_bus bus_idx, gpa_t addr,
 	r = __kvm_io_bus_write(vcpu, bus, &range, val);
 	return r < 0 ? r : 0;
 }
+EXPORT_SYMBOL_GPL(kvm_io_bus_write);
 
 /* kvm_io_bus_write_cookie - called under kvm->slots_lock */
 int kvm_io_bus_write_cookie(struct kvm_vcpu *vcpu, enum kvm_bus bus_idx,
@@ -3546,7 +3547,6 @@ static int __kvm_io_bus_read(struct kvm_vcpu *vcpu, struct kvm_io_bus *bus,
 
 	return -EOPNOTSUPP;
 }
-EXPORT_SYMBOL_GPL(kvm_io_bus_write);
 
 /* kvm_io_bus_read - called under kvm->slots_lock */
 int kvm_io_bus_read(struct kvm_vcpu *vcpu, enum kvm_bus bus_idx, gpa_t addr,
@@ -3568,7 +3568,6 @@ int kvm_io_bus_read(struct kvm_vcpu *vcpu, enum kvm_bus bus_idx, gpa_t addr,
 	return r < 0 ? r : 0;
 }
 
-
 /* Caller must hold slots_lock. */
 int kvm_io_bus_register_dev(struct kvm *kvm, enum kvm_bus bus_idx, gpa_t addr,
 			    int len, struct kvm_io_device *dev)
